### PR TITLE
First Implementation

### PR DESCRIPTION
**Pull request by pymike00**
First working "donation" functionalities implemented. 

Maybe a bit hacky.

Refactoring might be useful.

### CODE REVIEW

**Review by wagner-deoliveira — APPROVED** [review]
Looks great!

File: js/cheap.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  // TODO: check if Metamask is installed
+  const accounts = await ethereum.request({ method: "eth_requestAccounts" });
+  return accounts[0];
+}
+
+function getValue() {
+  // get the donation value inserted by the user
+  const amount = document.getElementById("donation-input").value;
+  if (isNaN(amount)) {
+    document.getElementById("donation-input").value =
```
Comment (wagner-deoliveira) [inline js/cheap.js:20]: Something has passed through my mind. Sometimes jquery is picky and to get the value there's (also) another way, which is $('#donation-input').val(). Just to have this in mind ;)

**Review by flyerman — APPROVED** [review]
👍 Neat code!

**Comment by flyerman** [comment]
I'm going to deploy the code on node.pool.cheap/gofundgeohot

File: js/cheap.js (hunk 1 above)
Comment (pymike00) [reply js/cheap.js:20]: Yes indeed I could have used jQuery too given that we have it here :) I have been full vanilla in such cases for a while as it's often been deprecated nowadays (e.g. bootstrap 5)

**Pull request merged**